documentation/src/main/asciidoc/ch02.asciidoc

```diff
@@ -750,7 +750,8 @@ optionally allowed attributes using the annotation `@SafeHtml.Tag`.
 
 |`@ScriptAssert(lang=, +
               script=, +
-              alias=)`
+              alias=, +
+              reportOn=)`
```

Comment:
I think it would be nice to add a sentence about thre `reportOn` option in the description below.